connector_algolia/views/se_backend_algolia.xml

```diff
@@ -4,50 +4,19 @@
     <record model="ir.ui.view" id="se_backend_algolia_form_view">
         <field name="name">se.backend.spec.abstract.form (in connector_search_engine)</field>
         <field name="model">se.backend.algolia</field>
+        <!--Mandatory to use the view when inherited (cfr product.template and product.product)-->
```

Comment:
@acsonefho s/Mandatory to use the view/Mandatory to use the mode/

A better explanation...

```Defining a view using the mode primary in conjunction with the inherit_id field allows to define a new view for the specified model by inheriting of an existing one defined for an other model. (by default mode value is extension```